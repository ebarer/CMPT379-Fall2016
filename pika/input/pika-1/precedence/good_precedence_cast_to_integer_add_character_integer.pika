exec {

  print [ ^p^ | int ] + 94, _n_.
  print [ ^"^ | int ] + 17, _n_.
  print [ ^O^ | int ] + 14, _n_.
  print [ ^S^ | int ] + 11, _n_.
  print [ ^L^ | int ] + 21, _n_.
  print [ ^p^ | int ] + 77, _n_.
  print [ ^I^ | int ] + 2, _n_.
  print [ ^t^ | int ] + 92, _n_.
  print [ ^1^ | int ] + 61, _n_.
  print [ ^1^ | int ] + 98, _n_.
  print [ ^g^ | int ] + 38, _n_.
  print [ ^=^ | int ] + 78, _n_.

  print 11 + [ ^5^ | int ], _n_.
  print 68 + [ ^g^ | int ], _n_.
  print 51 + [ ^8^ | int ], _n_.
  print 61 + [ ^f^ | int ], _n_.
  print 6 + [ ^O^ | int ], _n_.
  print 27 + [ ^}^ | int ], _n_.
  print 41 + [ ^k^ | int ], _n_.
  print 66 + [ ^Q^ | int ], _n_.
  print 69 + [ ^a^ | int ], _n_.
  print 40 + [ ^)^ | int ], _n_.
  print 40 + [ ^X^ | int ], _n_.
  print 43 + [ ^y^ | int ], _n_.

  print [ ^B^ | int ] + [ ^1^ | int ], _n_.
  print [ ^>^ | int ] + [ ^X^ | int ], _n_.
  print [ ^3^ | int ] + [ ^j^ | int ], _n_.
  print [ ^V^ | int ] + [ ^M^ | int ], _n_.
  print [ ^/^ | int ] + [ ^7^ | int ], _n_.
  print [ ^H^ | int ] + [ ^8^ | int ], _n_.
  print [ ^N^ | int ] + [ ^w^ | int ], _n_.
  print [ ^Y^ | int ] + [ ^Z^ | int ], _n_.
  print [ ^N^ | int ] + [ ^;^ | int ], _n_.
  print [ ^c^ | int ] + [ ^(^ | int ], _n_.
  print [ ^{^ | int ] + [ ^@^ | int ], _n_.
  print [ ^w^ | int ] + [ ^j^ | int ], _n_.

}

