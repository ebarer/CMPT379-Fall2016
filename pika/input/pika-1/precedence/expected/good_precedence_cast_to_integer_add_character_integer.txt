206
51
93
94
97
189
75
208
110
147
141
139
64
171
107
163
85
152
148
147
166
81
128
164
115
150
157
163
102
128
197
179
137
139
187
225
